backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/impl/HUPIItemProductDAO.java

```diff
@@ -75,6 +76,9 @@
 
 public class HUPIItemProductDAO implements IHUPIItemProductDAO
 {
+	private static final ISysConfigBL sysConfigBL = Services.get(ISysConfigBL.class);
+	private static final String SYSCONFIG_Enforce_M_HU_PI_Item_Product_ID = "de.metas.handlingunits.impl.HUPIItemProductDAO.Enforce_M_HU_PI_Item_Product_ID";
```

Comment:
move this sysconfig to de.metas.ui.web.quickinput.field.PackingItemProductFieldHelper and call it `de.metas.ui.web.quickinput.field.PackingItemProductFieldHelper.EnforcePrecisePricePerHUItemProduct`.